app/models/person.rb

```diff
@@ -4,7 +4,22 @@ class Person < VersionedModel
   has_many :profiles, dependent: :destroy
   has_many :moves, dependent: :destroy
 
+  has_one_attached :picture
+
   def latest_profile
     profiles.last
   end
+
+  def attach_picture(image_blob)
```

Comment:
was it not possible to use a StringIO object here? If not, I think an intention comment would be useful

Reply:
I think I got an error about the format, I think it's clear from the fact we handle an image here that it is a blob, Binary Large Object.